.github/workflows/dependabot_prs.yml

```diff
@@ -0,0 +1,65 @@
+name: DependaBot PRs
+
+on:
+  pull_request:
+
+permissions:
+  actions: read
+  checks: none
+  contents: write
+  deployments: none
+  issues: read
+  packages: none
+  pull-requests: write
+  repository-projects: none
+  security-events: none
+  statuses: none
+
+jobs:
+  handle_npm_update:
+    name: Handle NPM dependency updates for production packages
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.user.login == 'dependabot[bot]'
+    steps:
+      - name: Dependabot metadata
+        id: dependabot-metadata
+        uses: dependabot/fetch-metadata@v2
+      - uses: actions/checkout@v4
+        if: ${{ steps.dependabot-metadata.outputs.package-ecosystem == 'npm_and_yarn' }}
+        with:
+          ref: ${{ github.head_ref }}
+          submodules: false
+          lfs: false
+          persist-credentials: false
+      - name: Prepare Git
+        if: ${{ steps.dependabot-metadata.outputs.package-ecosystem == 'npm_and_yarn' }}
+        run: |
+          cat <<- EOF > $HOME/.netrc
+            machine github.com
+            login innocraft-automation
+            password $CUSTOM_ACCESS_TOKEN
+          EOF
+          chmod 600 $HOME/.netrc
+
+          git config --global user.email "innocraft-automation@users.noreply.github.com"
+          git config --global user.name "innocraft-automation"
+          git remote set-url origin https://x-access-token:${{ secrets.CUSTOM_ACCESS_TOKEN }}@github.com/$GITHUB_REPOSITORY
```

Comment:
In the composer action we have
```
git config --global user.email "$GITHUB_ACTOR@users.noreply.github.com"
git config --global user.name "$GITHUB_ACTOR"
git remote set-url origin https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/$GITHUB_REPOSITORY
```
would this be changed to that?

Reply:
I guess the actor is who runs the action, and the automation is a neutral user account, so it might be better to keep this way of doing it in both actions?

Reply:
This is on purpose. Using the actor of the action to commit something prevents the commit from triggering further actions (like tests). In this case we want actions to be triggered. But we should consider to use the automation user also in other actions.